python/hail/docs/tutorials/09-matrixtable.ipynb

```diff
@@ -0,0 +1,476 @@
+    "\n",
+    "Since rating is very sparse, it might be reasonable to store in this way in so-called \"coordinate form\" where each rating is a row of a table.\n",
+    "\n",
+    "However, the per-row overhead becomes untenable when working with large, dense datasets like sequence data!  That's why we built `MatrixTable`, a 2-dimensional generalization of `Table`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# MatrixTable Anatomy\n",
+    "\n",
+    "Remember `Table` had two kinds of fields: globals and row fields.\n",
+    "\n",
+    "`MatrixTable` has four kinds of fields:\n",
+    " - the globals\n",
+    " - the row fields\n",
+    " - the column fields\n",
+    " - the entry fields"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   },
+   "source": [
+    "Row fields are fields that are stored once per row.  These are like INFO fields in a VCF.\n",
+    "\n",
+    "Column fields are stored once per column.  These have no analog in VCF files, but for sequence data, column fields can store sample metadata, phenotypes, etc.\n",
+    "\n",
+    "And, of course, entry fields are like the FORMAT fields in a VCF."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Importing and Reading\n",
+    "\n",
+    "Like tables, matrix tables can be *imported* from a variety of formats: VCF, (B)GEN, PLINK, TSV, etc.  Matrix tables can also be *read* from a \"native\" matrix table format.  Let's read a sample of prepared 1KG data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import hail as hl\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn\n",
+    "seaborn.set()\n",
+    "%matplotlib inline\n",
+    "\n",
+    "hl.utils.get_1kg('data/')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "mt = hl.read_matrix_table('data/1kg.mt')\n",
+    "mt.describe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "There are a few things to note:\n",
+    " - There is a single column field `s`.  This is the sample ID from the VCF.  Note, it is the column key.\n",
+    " - There is a compound row key: `locus` and `alleles`.  \n",
+    "     - `locus` has type `locus<GRCh37>`\n",
+    "     - `alleles` has type `array<str>`\n",
+    " - GT has type `call`.  That's a genotype call!\n",
+    " - There is another key type: the partition key. Don't worry about this right now."
```

Comment:
maybe change to: "the partition key, which relates to how the rows are grouped into partitions for processing. Don't worry about this for now."